src/components/BookForm.js

```diff
@@ -0,0 +1,15 @@
+// components/BookForm.js
+
+const BookForm = () => (
+  <div className="form">
+    <h2>Add New Book</h2>
+    <form>
+      <input type="text" />
+      <input type="text" />
+      <input type="text" />
```

Comment:
- [ ] [OPTIONAL] The form should contain only two inputs and a button, kindly consider adding a placeholder text for the two inputs. the first can be the "book title", and the second "author"